src/patternfly/components/Select/select-menu-filter.hbs

```diff
@@ -0,0 +1,4 @@
+{{#> select-menu-group select-menu-group--modifier="pf-m-padding"}}
```

Comment:
Should we follow what is in the context selector component? It has its own element wrapper for the input. The advantage of this is that it is specific for adding an input in it. While the modifier that you're applying to the `pf-c-select__menu-group` keeps it flexible, it may mean that users use it for the wrong reasons other than what its meant for.

<img width="324" alt="Screen Shot 2020-02-11 at 1 49 39 PM" src="https://user-images.githubusercontent.com/20118816/74268226-83ba6f80-4cd5-11ea-911c-73d14e6abdcc.png">

Reply:
Probably should do that. If in the future we want to add `.pf-c-select__menu-group.pf-m-padding` in the future, we can.